v3/library-after-failure.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+set -e
+
+# Necessary to stop pull requests from forks from running.
+if [ "$TRAVIS_SECURE_ENV_VARS" == "true" ]; then
+  node ./node_modules/@blackbaud/skyux-builder-config/scripts/visual-failures.js
+else
+  echo -e "Ignoring script."
```

Comment:
This message gets printed directly after the script reference in the logs:
<img width="768" alt="screen shot 2018-09-06 at 10 58 13 am" src="https://user-images.githubusercontent.com/12497062/45165910-c6644800-b1c3-11e8-9027-f8928ab9408b.png">

Though, it might be useful to explain why the script is being ignored.